templates/submit-with-summary.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'layouts/_submit.html' %}
+
+{% block summary %}
+  <div class="u-mb-m">
+    {% include 'partials/summary/summary.html' %}
+  </div>
+{%  endblock summary %}
+
+{% block submit_button %}
+  <p class="u-mt-m">
+    {{
+      onsButton({
+        "text": content.submit_button,
+        "submitType": 'timer',
+        "classes": "btn--loader js-btn-submit",
+        "attributes": {
+          "data-qa": "btn-submit",
+          "data-loading-msg": _("Submitting") + "&hellip;"
+        }
+      })
+    }}
+  </p>
+{% endblock submit_button %}
```

Comment:
This was wrong actually, it should have been different for `hub` vs `submit`. However, I now use the same config for all of them. I used the hub's definition since it took into account displaying a guidance above the button which now applies to submit pages